src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -468,8 +472,21 @@ internal void SetRedoButtonStatus(bool enabled)
 
     internal void NotifyFreebuild(object freebuilding)
```

Comment:
Shouldn't this take in a bool? Instead of just casting it anyway in the method body?